graph/articles/coreTypes.md

```diff
@@ -0,0 +1,97 @@
+# Core Types
+
+## Overview
+
+Types exist in graph which are highly-connected/central to the graph ecosystem. Often, these types are the position of containing many structural properties relevant to other APIs, because they are connected to many entities in the graph.
```

Comment:
```suggestion
Types exist in graph which are highly-connected/central to the graph ecosystem. Often, these types are in the position of containing many structural properties relevant to other APIs, because they are connected to many entities in the graph.
```